sequencer/src/persistence.rs

```diff
@@ -1145,6 +1157,41 @@ mod persistence_tests {
             .is_err());
     }
 
+    async fn assert_events<P: TestablePersistence>(
```

Comment:
nit: maybe say something about what we're asserting? by default in rust `assert!` just means checking that something is true.

```suggestion
    async fn assert_events_eq<P: TestablePersistence>(
```